src/modules/files/controllers/file.controller.ts

```diff
@@ -26,20 +30,31 @@ export class FileController {
     return new CreateFileResponseTransformer().item(file, uploadUrl)
   }
 
-  @Post('/:file/confirm-upload')
-  @ApiResponse(confirmFileUploadApiResponse)
+  @Delete('/:fileUuid')
+  @ApiResponse(removeFileResponse)
+  @Permissions(Permission.FILE_DELETE)
+  public async removeFile (
+    @UuidParam('fileUuid') fileUuid: string
+  ): Promise<void> {
+    await this.fileFlowService.remove(fileUuid)
+  }
+
+  @Post('/:fileUuid/confirm-upload')
+  @ApiResponse(confirmFileUploadResponse)
   @HttpCode(200)
-  async confirmFileUpload (
-    @Param('file', ParseUUIDPipe) fileUuid: string
+  @Permissions(Permission.FILE_UPDATE)
```

Comment:
```suggestion
  @Permissions(Permission.FILE_CREATE)
```

Part of the create flow